spec/models/invitation_spec.rb

```diff
@@ -78,5 +78,21 @@
         end
       end
     end
+
+    # Regression test for AO3-6094
+    context "Sending an invitation to a valid email" do
+      let(:invite) { build(:invitation, invitee_email: "foo@example.com") }
+
+      it "succeeds and 'sent_at' is set" do
+        expect(invite.save).to be_truthy
+        expect(invite.id).not_to be_nil
+
+        # Reload invite to make sure that change to invite.sent_at is actually persisted
+        # on the database (not just on the 'invite' object currently being created).
+        reloaded_invite = Invitation.find_by(id: invite.id)
```

Comment:
You can use [`reload`](https://api.rubyonrails.org/v6.1.4/classes/ActiveRecord/Persistence.html#method-i-reload) for this. We do that in a lot of tests so you don't need to comment on its use either.

Reply:
Heh, I wasn't expecting something so convenient! Thanks for pointing ``reload`` out :)